instrumentation/github.com/gorilla/mux/otelmux/mux.go

```diff
@@ -18,12 +18,15 @@ import (
 	"encoding/json"
 	"fmt"
 	"io"
+	"mime"
 	"net/http"
 	"os"
+	"strings"
 	"sync"
 
 	"github.com/felixge/httpsnoop"
 	"github.com/gorilla/mux"
+	"golang.org/x/exp/slices"
 
 	obfuscator "github.com/helios/go-sdk/data-obfuscator"
```

Comment:
Can we add this logic to the obfuscator (perhaps rename it) to avoid all this code duplication?